src/Illuminate/Routing/Router.php

```diff
@@ -1089,6 +1089,27 @@ public function currentRouteUses($action)
     }
 
     /**
+     * Determine if the current action matches a given action.
+     * 
+     * This take in account the namespace of the application.
+     * 
+     * @param string $action
+     *
+     * @return bool
+     */
+    public function currentActionUses($action)
+    {
+        $currentAction = $this->getCurrentRoute()->getAction();
```

Comment:
Please add an extra linebreak. We don't yet have a StyleCI fixer to enforce this.